pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -1254,712 +1240,659 @@ var _ = Describe("ProjectService", func() {
-
-			When("any of time based paramaters is set to 0", func() {
-				JustBeforeEach(func() {
-					projectService.Labels = composego.Labels{
-						config.LabelWorkloadReadinessProbeType:    config.ProbeTypeExec.String(),
-						config.LabelWorkloadReadinessProbeTimeout: "0",
-					}
-				})
-
-				It("logs and returns error", func() {
-					_, err := projectService.readinessProbe()
-					Expect(err).To(HaveOccurred())
-					Expect(err).To(MatchError("Readiness probe misconfigured"))
+					Expect(projectService.Extensions).To(Equal(extensions))
 
-					assertLog(logrus.ErrorLevel,
-						"Readiness probe misconfigured",
-						map[string]string{},
-					)
+					lp, err := projectService.LivenessProbe()
+					Expect(err).NotTo(HaveOccurred())
+					Expect(lp.HTTPGet).NotTo(BeNil())
+					Expect(lp.HTTPGet.Path).To(Equal(""))
 				})
 			})
 		})
 	})
 
-	Describe("readinessProbeTCP", func() {
+	Describe("livenessProbeTCP", func() {
 		When("defined via labels", func() {
 
 			Context("and supplied as string port", func() {
 				port := "8080"
 
 				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadReadinessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadReadinessProbeTCPPort: port,
-					}
+					k8sconf.Workload.LivenessProbe.Type = config.ProbeTypeTCP.String()
+					k8sconf.Workload.LivenessProbe.TCP.Port = 8080
 				})
 
 				It("returns label value", func() {
-					p, err := projectService.readinessProbe()
+					p, err := projectService.LivenessProbe()
 					Expect(err).To(Succeed())
 					Expect(p.TCPSocket.Port.String()).To(Equal(port))
 				})
 			})
 
 			Context("and empty port", func() {
 				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadReadinessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadReadinessProbeTCPPort: "",
-					}
-				})
-
-				It("returns an error", func() {
-					p, err := projectService.readinessProbe()
-					Expect(err).To(HaveOccurred())
-					Expect(err.Error()).To(Equal(fmt.Sprintf("%s needs to be a number", config.LabelWorkloadReadinessProbeTCPPort)))
-					Expect(p).To(BeNil())
-				})
-			})
-
-			Context("with NaN port", func() {
-				BeforeEach(func() {
-					labels = composego.Labels{
-						config.LabelWorkloadReadinessProbeType:    config.ProbeTypeTCP.String(),
-						config.LabelWorkloadReadinessProbeTCPPort: "asds",
-					}
+					k8sconf.Workload.LivenessProbe.Type = config.ProbeTypeTCP.String()
+					k8sconf.Workload.LivenessProbe.TCP.Port = 0
 				})
 
 				It("returns an error", func() {
-					_, err := projectService.readinessProbe()
-					Expect(err).NotTo(Succeed())
-					Expect(err.Error()).To(ContainSubstring("%s needs to be a number", config.LabelWorkloadReadinessProbeTCPPort))
+					p, err := projectService.LivenessProbe()
+					Expect(err).NotTo(HaveOccurred())
+					Expect(p.TCPSocket.Port.String()).To(Equal("0"))
 				})
 			})
 
 			Context("and no port label", func() {
```

Comment:
It's not a label anymore so let's change the wording here too.

Reply:
I'll review all of this once we have values being read and labels completely removed